scripts/find-node.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -e
+
```

Comment:
Fixes https://github.com/react-native-community/upgrade-support/issues/138#issuecomment-798788120

```suggestion

unset npm_config_prefix

```